src/flags.rs

```diff
@@ -206,6 +214,8 @@ impl<'a> From<&'a str> for Block {
             "user" => Block::User,
             "group" => Block::Group,
             "size" => Block::Size,
+            "size_value" => Block::SizeValue,
```

Comment:
These are not added to `app.rs` in the cli arg parser.

Reply:
At the moment, I need the `size_value` for the left padding calculation but I have removed the `unit_value`.

I currently work to add the left padding feature into the term_grid library. Once it's done we should be able to remove all the padding logic. I hope this will be my next PR

Reply:
> @Peltoche The code looks good. But there seems to be mess when using `-l` with `--tree`. It seem to only respect `-l`


Fixed